tests/test_scripts.py

```diff
@@ -2693,6 +2694,17 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting():
+    infile = utils.get_temp_filename('count.ct.gz')
+    indir = os.path.dirname(infile)
+    shutil.copyfile(utils.get_test_data('count.ct.gz'), infile)
+    assert os.path.exists(infile), infile
+    try:
+        test_ct = khmer.load_counting_hash(infile)
+    except IOError:
+        raise Exception("Can't load a counting table with bigcount set")
```

Comment:
Exceptions should not be raised in any of the tests; this prevents other tests from running even if `stop = FALSE` is set.

Try `assert 0, "Can't load a counting table with bigcount set"` instead.